src/main/java/by/byshnev/repositories/ArticleRepository.java

```diff
@@ -0,0 +1,9 @@
+package by.byshnev.repositories;
+
+import by.byshnev.models.Article;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+
+
+public interface ArticleRepository extends CrudRepository<Article, Long> {
```

Comment:
Decouple data access layer from other layers of your app with DAO interface. You can have multiple implementations of the dao like this CrudRepository or implementation that fetches articles from json file. Use it in your service. Service should not care what implementation it is using